db/models/articles-model.js

```diff
@@ -16,3 +16,21 @@ exports.fetchArticlesById = (article_id) => {
         }
     })   
 };
+
+exports.fetchAllArticleComments = (article_id) => {
+    return db
+    .query(
+        `SELECT * FROM comments
+         WHERE article_id = $1
+         ORDER BY created_at DESC;`, [article_id])
+    .then((result)=> {
+        const { rows } = result
+        if (rows.length === 0) {
+            return Promise.reject({ 
+                status: 404, message: 'article_id does not exist'
+        })
```

Comment:
There will need to be some functionality added to check that an article exists but has no comments 👍

Reply:
completed